install_config/upgrading/manual_upgrades.adoc

```diff
@@ -744,6 +974,119 @@ IMAGE_VERSION=<tag>
 +
 Replace `<tag>` with `{latest-tag}` for the latest version.
 
+. Configure Dnsmasq. Because of changes made in {product-title} 3.6, you will need to
+perform the following steps to configure dnsmasq as part of the upgrade. See DNS Changes
+under xref:../../release_notes/ocp_3_6_release_notes.adoc#ocp-36-notable-technical-changes[Notable Technical Changes]
+in the {product-title} 3.6 Release Notes.
++
+.. Download the following script:
++
+----
+# wget https://raw.githubusercontent.com/openshift/openshift-ansible/release-3.6/roles/openshift_node_dnsmasq/files/networkmanager/99-origin-dns.sh -O /etc/NetworkManager/dispatcher.d/99-origin-dns.sh
+# chmod 755 /etc/NetworkManager/dispatcher.d/99-origin-dns.sh
+----
++
+This command downloads the *99-origin-dns.sh* script and copies the script to the *_/etc/NetworkManager/dispatcher.d/_* directory.
+This script configures pods to use the node IP address as resolver. Using 127.0.0.1 inside a pod would fail.
+
++
+.. Create a *_/etc/origin/node/node-dnsmasq.conf_*  node configuration file.
++
+----
+server=/in-addr.arpa/127.0.0.1
+server=/cluster.local/127.0.0.1
+----
++
+Sample *_node-dnsmasq.conf_* file:
++
+[source, bash]
+----
+$ cat /etc/origin/node/node-dnsmasq.conf
+  server=/in-addr.arpa/127.0.0.1
+  server=/cluster.local/127.0.0.1
+----
+
+.. Edit the *_/etc/dnsmasq.d/origin-dns.conf_* file as follows:
++
+----
+no-resolv
+domain-needed
+no-negcache
+max-cache-ttl=1
+enable-dbus
+bind-interfaces
+listen-address=<node_ip_address> <1>
+----
+<1> This is the IP address of the node host.
++
+Sample *_origin-dns.conf_* file:
++
+[source, bash]
+----
+$ cat /etc/dnsmasq.d/origin-dns.conf
+  no-resolv
+  domain-needed
+  no-negcache
+  max-cache-ttl=1
+  enable-dbus
+  bind-interfaces
+  listen-address=<node_ip_address>
+----
+
+.. Edit the *_/etc/dnsmasq.d/origin-upstream-dns.conf_* file as follows:
++
+----
+server=<dns_server1_ip_address>
+server=<dns_server2_ip_address>
```

Comment:
Please make sure we always us the same place-holder. In here we now have `<dns_server2_ip_address>` and `<upstream-dns-server-2>` for the upstream DNS server. Please align them to avoid any confusion.

It potentially could also help to create an example, showing what is meant with `<upstream-dns-server-2>` to make it clear for the customer. So for example, highlighting 8.8.8.8 or 9.9.9.9 as an example upstream DNS server.